src/pages/home/HeaderView.tsx

```diff
@@ -334,7 +334,7 @@ function HeaderView({report, personalDetails, parentReport, parentReportAction,
                                     )}
                                 </PressableWithoutFeedback>
                                 <View style={[styles.reportOptions, styles.flexRow, styles.alignItemsCenter]}>
-                                    {isTaskReport && !isSmallScreenWidth && ReportUtils.isOpenTaskReport(report) && <TaskHeaderActionButton report={report} />}
+                                    {isTaskReport && !isSmallScreenWidth && ReportUtils.isOpenTaskReport(report, parentReportAction) && <TaskHeaderActionButton report={report} />}
```

Comment:
@dukenv0307 this change is enough for me. No need changes in ReportScreen. Am I missing anything?

https://github.com/Expensify/App/assets/108292595/be54f8f7-dcff-41ea-907b-58087732d4c0

Reply:
Ah yes, but the current change will cause performance regression on ReportScreen.
I think what I reported is minor and can be out of scope if it's not straightforward fix.